lib/portlayer/exec/handle.go

```diff
@@ -39,12 +39,19 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/session"
 )
 
+// Resources describes the resource allocation for the containerVM
+type Resources struct {
+	NumCPUs  int64
```

Comment:
It's a choice about when/where you want to do the cast. It comes in from Docker CLI as an int64 but govmomi requires int32, so it's basically just an arbitrary decision on where you cast the int.